ARK Server Manager/Lib/Model/ServerProfile.cs

```diff
@@ -1889,5 +1896,25 @@ public void ResetOverrideMaxExperiencePointsDino()
         {
             OverrideMaxExperiencePointsDino = GameData.DEFAULT_MAX_EXPERIENCE_POINTS_DINO;
         }
+
+        // section reset methods
+        public void ResetChatAndNotificationSection()
+        {
+            EnableGlobalVoiceChat = GameData.DEFAULT_ENABLE_GLOBAL_VOICE_CHAT;
```

Comment:
Having them as named constants is nice and generally a good pattern to follow.  It keeps all of the game-related values in a single place which means it's easy for us to see and update them.